test/framework/logging.go

```diff
@@ -158,6 +160,16 @@ func getLogFile(namespace, filename string) string {
 	return getLogFolder(namespace) + "/" + filename + logSuffix
 }
 
+// InitAndGetLogFolder setup and return the main log folder
+func InitAndGetLogFolder(subFolder string) (string, error) {
+	logFolder = defaultLogFolder
```

Comment:
This looks like a method side effect for me.
From the method signature I would expect that log folder is just retrieved and initialized. I would prefer separate method for setting log folder.
Also log folder initialization may be a separate method.

Reply:
updated

Reply:
Thanks